packages/fxa-auth-server/bin/key_server.js

```diff
@@ -135,6 +141,27 @@ async function run(config) {
 
   const profile = require('../lib/profile/client')(log, config, statsd);
   Container.set(ProfileClient, profile);
+
+  if (
+    config.contentful &&
+    config.contentful.cdnApiUri &&
+    config.contentful.graphqlUrl &&
+    config.contentful.apiKey &&
+    config.contentful.spaceId &&
+    config.contentful.environment
+  ) {
+    const contentfulClient = new ContentfulClient({
+      cdnApiUri: config.contentful.cdnApiUri,
+      graphqlApiUri: config.contentful.graphqlUrl,
+      graphqlApiKey: config.contentful.apiKey,
+      graphqlSpaceId: config.contentful.spaceId,
+      graphqlEnvironment: config.contentful.environment,
+    });
+    const contentfulManager = new ContentfulManager(contentfulClient);
+    const capabilityManager = new CapabilityManager(contentfulManager);
+    Container.set(CapabilityManager, capabilityManager);
```

Comment:
Since having this exist is required for the `StripeHandler`, we should require these arguments if we setup the StripeHandler farther down. e.g. in the block where we create the `StripeHelper`, first verify these config args are present, then make and set the `CapabilityManager`.